client/src/use/useStyling.ts

```diff
@@ -35,82 +55,154 @@ export default function useStyling({ markChangesPending }: UseStylingParams) {
     strokeWidth: 3,
     opacity: 1.0,
     color: 'type',
+    fill: false,
   };
   const selected: Style = {
     ...standard,
     color: vuetify.preset.theme.themes.dark.accent as string,
     strokeWidth: 5,
     opacity: 1.0,
+    fill: false,
   };
   const disabled: Style = {
     ...standard,
     color: 'type',
     strokeWidth: 0.5,
     opacity: 0.45,
+    fill: false,
   };
   // Colors provided for the different Types
   const stateStyling: StateStyles = { standard, selected, disabled };
-  const typeColors = [
-    colors.red.accent1,
-    colors.yellow.darken3,
-    colors.purple.lighten3,
-    colors.green.lighten3,
-    colors.yellow.lighten3,
-    colors.purple.darken3,
-    colors.green.darken3,
-  ];
+  const typeColors = [];
 
-  function loadTypeColors(list?: Record<string, string>) {
-    if (list) {
+
+  const numColors = 12; //We can up the number of colors but they will become similar;
+  for (let i = 0; i < numColors; i += 1) {
```

Comment:
A comment here explaining that a color palette is being generated would be helpful.   Extracting this to a separate function might be even better.

Reply:
I didn't know if I should put it in utils.js or leave it here.  I think it makes sense having the generated color function here as well.